components/builder-web/package-lock.json

```diff
@@ -4302,6 +4307,256 @@
         "extend": "3.0.1"
       }
     },
+    "http-proxy-middleware": {
```

Comment:
Are we including this in the prod package as well?  Not clear whether this is dev only..

Reply:
Nope, it's a `devDependency`, so it's only needed in dev.